server/svix-server/src/core/idempotency.rs

```diff
@@ -375,7 +377,7 @@ mod tests {
     ) -> (JoinHandle<()>, String, Arc<Mutex<usize>>) {
         dotenv::dotenv().ok();
 
-        let cache = cache::memory::new();
+        let store = SharedStore::MemorySharedStore(MemoryKv::new());
```

Comment:
This feels very redundant, if we already have SharedStore::MemorySharedStore, why not have it create a new memory kv on its own (internally)?